src/less/overwrites/_forms.less

```diff
@@ -23,7 +23,7 @@ label{
 }
 
 textarea{
-  resize: none;
+  // resize: none;
```

Comment:
@saadmhybris as @taifunhaeberle pointed out you need to use a rule for vertical resize only.

Please try using: 

resize: vertical;